src/docs/index.html

```diff
@@ -1,19 +1,85 @@
 <!doctype html>
 <html lang="en">
-	<head>
+	<head
+		prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb# article: http://ogp.me/ns/article#"
+	>
```

Comment:
Not sure this is required, it might have been part of a more legacy approach to Open Graph — the links currently 404, and appending the `head` with additional metadata like this isn't mentioned on pages like [Facebook's guide to adding opengraph tags for Facebook](https://developers.facebook.com/docs/sharing/webmasters/#markup). But I also don't think it's hurting anything, so I'll leave it in.